### ISSUE TITLE
SplitMap with geojson shapes

### ISSUE THREAD

**Issue by mangecoeur** [jupyter-widgets/ipyleaflet#179]
It seems SplitMap will only accept tilemaps as left/right layers, so I can't figure out how to add geojson shapes to the splitMap - is this supported at all?

**Comment by SylvainCorlay** [comment]
Indeed, at the moment, splitmap only takes tilemaps.

However, adding support for arbitrary layers should be doable.

Are you interested in taking a stab at it?

**Comment by martinRenou** [comment]
Closed #296. SplitMapControl nows supports lists of arbitrary layers for the left and right.

**Comment by deeplook** [comment]
I'm trying to use this like the following, but I'm not sure this works. Is this the way it's intended to work?

```python
geo_json = GeoJSON(data=requests.get(<someURL>).json())
right_layer = basemap_to_tiles(basemaps.OpenStreetMap["Mapnik"])
left_layer = basemap_to_tiles(basemaps.OpenStreetMap["BlackAndWhite"])
control = SplitMapControl(
    left_layer=[right_layer, geo_json],
    right_layer=right_layer)
```
